src/leiningen/new/db.clj

```diff
@@ -89,6 +94,25 @@
                                         ['com.google.guava/guava "25.1-jre"]
                                         ['io.rkn/conformity "0.5.1"]])))])
 
+(defn add-crux [[assets options]]
+  [(into assets crux-files)
+   (-> options
+       (assoc
+         :crux true
+         :db-connection true
```

Comment:
I wasn't sure what `:db-connection` did, couldn't see it being used but left it in for consistency with the other db profiles